ocaml-lsp-server/src/ocaml_lsp_server.ml

```diff
@@ -863,8 +864,50 @@ let stream_of_channel : Lsp.Cli.Channel.t -> _ = function
     let sockaddr = Unix.ADDR_INET (Unix.inet_addr_loopback, port) in
     socket sockaddr
 
+(* Merlin uses [Sys.command] to run preprocessors and ppxes. We provide an
+   alternative version using spawn for unixes.*)
```

Comment:
nit:
can we capitalize "spawn", or even change it to "Spawn library", so that it's more straightforward?